--- rubygem-passenger-3.0.5/build/config.rb.flags~	2011-03-29 02:34:17.000000000 +0200
+++ rubygem-passenger-3.0.5/build/config.rb	2011-03-29 02:35:14.307287699 +0200
@@ -33,10 +33,11 @@ end
 # Extra compiler flags that should always be passed to the C/C++ compiler.
 # Should be included last in the command string, even after PlatformInfo.portability_cflags.
 EXTRA_CXXFLAGS = "-Wall #{OPTIMIZATION_FLAGS}".strip
+EXTRA_CXXFLAGS << ENV['CXXFLAGS']
 
 # Extra linker flags that should always be passed to the linker.
 # Should be included last in the command string, even after PlatformInfo.portability_ldflags.
-EXTRA_LDFLAGS  = ""
+EXTRA_LDFLAGS  = ENV['LDFLAGS']
 
 # Whether to use the vendored libev or the system one.
-USE_VENDORED_LIBEV = boolean_option("USE_VENDORED_LIBEV", true)
\ No newline at end of file
+USE_VENDORED_LIBEV = boolean_option("USE_VENDORED_LIBEV", true)
